net/qpid-proton/Portfile

```diff
@@ -19,14 +18,16 @@ homepage            https://qpid.apache.org
 
 PortGroup           github 1.0
 
-github.setup        apache qpid-proton 0.29.0
-revision            1
+github.setup        apache qpid-proton 0.31.0
+revision            0
```

Comment:
These lines should be moved to the top of the file. That way the homepage won't be overridden by the GitHub portgroup.